### ISSUE TITLE
Problems with current libtool *.la scheme

### ISSUE THREAD

**Issue by jpereira** [FreeRADIUS/freeradius-server#1433]
The current libfreeradius-radius.la scheme is a unrecognized archive. it's different of the documentation https://autotools.io/libtool/lafiles.html

```
[jpereira@jpereira-desktop mod_auth_radius.git]$ make
apxs -c -I/opt/radius-3.1.x/include -DHAVE_PTHREAD_H -DWITH_THREADS=1 -L/opt/radius-3.1.x/lib/ -Wl,-R/opt/radius-3.1.x/lib/ -lfreeradius-radius  mod_auth_radius.c
/usr/share/apr-1.0/build/libtool --silent --mode=compile --tag=disable-static x86_64-linux-gnu-gcc -std=gnu99 -prefer-pic -pipe -g -O2 -fstack-protector-strong -Wformat -Werror=format-security  -D_FORTIFY_SOURCE=2   -DLINUX -D_REENTRANT -D_GNU_SOURCE  -pthread  -I/usr/include/apache2  -I/usr/include/apr-1.0   -I/usr/include/apr-1.0 -I/usr/include -I/opt/radius-3.1.x/include -DHAVE_PTHREAD_H -DWITH_THREADS=1  -c -o mod_auth_radius.lo mod_auth_radius.c && touch mod_auth_radius.slo
/usr/share/apr-1.0/build/libtool --silent --mode=link --tag=disable-static x86_64-linux-gnu-gcc -std=gnu99 -Wl,--as-needed -Wl,-Bsymbolic-functions -Wl,-z,relro -Wl,-z,now    -o mod_auth_radius.la -R/opt/radius-3.1.x/lib/  -L/opt/radius-3.1.x/lib/ -lfreeradius-radius -rpath /usr/lib/apache2/modules -module -avoid-version    mod_auth_radius.lo
libtool: link: `/opt/radius-3.1.x/lib//libfreeradius-radius.la' is not a valid libtool archive
apxs:Error: Command failed with rc=65536
.
Makefile:25: recipe for target '.libs/mod_auth_radius.so' failed
make: *** [.libs/mod_auth_radius.so] Error 1
[jpereira@jpereira-desktop mod_auth_radius.git]$
```

```
[jpereira@jpereira-desktop mod_auth_radius.git]$ cat /opt/radius-3.1.x/lib/libfreeradius-radius.la 
/opt/radius-3.1.x/lib
[jpereira@jpereira-desktop mod_auth_radius.git]$
```

btw, sure! works fine when i use explicit library like: ```-l:libfreeradius-radius.so```

```
[jpereira@jpereira-desktop mod_auth_radius.git]$ make
apxs -c -I/opt/radius-3.1.x/include -DHAVE_PTHREAD_H -DWITH_THREADS=1 -L/opt/radius-3.1.x/lib/ -Wl,-R/opt/radius-3.1.x/lib/ -l:libfreeradius-radius.so  mod_auth_radius.c
/usr/share/apr-1.0/build/libtool --silent --mode=compile --tag=disable-static x86_64-linux-gnu-gcc -std=gnu99 -prefer-pic -pipe -g -O2 -fstack-protector-strong -Wformat -Werror=format-security  -D_FORTIFY_SOURCE=2   -DLINUX -D_REENTRANT -D_GNU_SOURCE  -pthread  -I/usr/include/apache2  -I/usr/include/apr-1.0   -I/usr/include/apr-1.0 -I/usr/include -I/opt/radius-3.1.x/include -DHAVE_PTHREAD_H -DWITH_THREADS=1  -c -o mod_auth_radius.lo mod_auth_radius.c && touch mod_auth_radius.slo
/usr/share/apr-1.0/build/libtool --silent --mode=link --tag=disable-static x86_64-linux-gnu-gcc -std=gnu99 -Wl,--as-needed -Wl,-Bsymbolic-functions -Wl,-z,relro -Wl,-z,now    -o mod_auth_radius.la -R/opt/radius-3.1.x/lib/  -L/opt/radius-3.1.x/lib/ -l:libfreeradius-radius.so -rpath /usr/lib/apache2/modules -module -avoid-version    mod_auth_radius.lo
[jpereira@jpereira-desktop mod_auth_radius.git]$
```

**Comment by arr2036** [comment]
Hmm i'll leave @alandekok to comment on that one.  I'm not sure they're proper libtool archive files.

**Comment by alandekok** [comment]
They're not valid libtool files. They probably shouldn't be installed. Instead just the .so and .a files should be installed.

**Comment by jpereira** [comment]
@alandekok Currently the *.a its already installed. btw, I fix the jlibtool in #1435
